java/java-ranger-regression/schedule_eqchk/TestSchedule2.java

```diff
@@ -0,0 +1,57 @@
+import org.sosy_lab.sv_benchmarks.Verifier;
+
+import java.util.Objects;
+
+public class TestSchedule2 {
+    void testHarness(Main v, int in0, int in1, int in2) {
+        in0 = Verifier.nondetInt();
+        in1 = Verifier.nondetInt();
+        in2 = Verifier.nondetInt();
+        Process[] outSPF = SPFWrapper(v, in0, in1, in2);
```

Comment:
You are right in pointing out that this equivalence check is the same as calling the testFunction twice and is unnecessarily verbose. I've left the extra stack frames in case a verifier would like to use the extra information in the stack frame to figure out which half of this benchmark it are looking at. This information was useful to Java Ranger for debugging purposes. If the extra fluff isn't hurting readability much, I would like to leave the extra stack frames in.

Reply:
From my point of view, this kind of extra stack frames would not be in real software. Hence, as SV-Comp benchmarks aim for realistic software, I think these frames should not be in the contribution, at least not as part of the test case.

Anyhow, maybe @peterschrammel or @dbeyer have a third opinion for this case.

Reply:
Alrighty, I've taken out the unnecessary stack frames too.